packages/typescript-estree/tests/lib/semanticInfo-singleRun.test.ts

```diff
@@ -195,7 +195,7 @@ describe('semanticInfo - singleRun', () => {
      * Single run because of process.argv
      */
     const originalProcessArgv = process.argv;
-    process.argv = ['', 'node_modules/.bin/eslint', ''];
+    process.argv = ['', path.normalize('node_modules/.bin/eslint'), ''];
```

Comment:
Heh, is this for Windows? If so, thanks! 😄

Reply:
yes it is :)